src/noted-objects-analysis/members-parser.js

```diff
@@ -247,7 +247,7 @@ function setMemberDefinitions(definition, comment, modifiers, constructor = fals
 
     tail = `(${outputParams(params)})`;
 
-    if (constructor === false && name !== 'constructor') {
+    if (constructor === false && name !== 'constructor' && name[0] !== 'constructor') {
```

Comment:
I assume name can be array here?

Reply:
Would it name be ever undefined causing crash here?

Reply:
Yes, it appears that name is an array when the function is a class function.  You can output to console to see.

Reply:
I guess name defaults to an empty string so it is all good here.